packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -0,0 +1,4603 @@
+openapi: 3.0.3
+info:
+  title: PDND Interoperability API
+  x-summary: PDND Interoperability API
+  version: "2.0.0"
+  contact:
+    name: API Support
+    url: "http://www.example.com/support"
+    email: support@example.com
+  termsOfService: "http://swagger.io/terms/"
+servers:
+  - url: "https://api.interop.pagopa.it/v2"
+    description: Production
+security:
+  - bearerAuth: []
+tags:
+  - name: clients
+    description: Clients routes
+    externalDocs:
+      description: Find out more
+      url: "http://swagger.io"
+  - name: attributes
+    description: Attribute routes
```

Align these descriptions, either all singular or plural, no strong opinion :)

```suggestion
    description: Attributes routes
```